workflow/read.md

```diff
@@ -0,0 +1,125 @@
+
+- [Test](http://test-tenement-museum.pantheonsite.io/) - Not visible for the client.
+- [Develop](http://test-tenement-museum.pantheonsite.io/) - Visible for the client.
+- [Production](http://test-tenement-museum.pantheonsite.io/) - Live site.
+
+## Built With (and versions)
+<!-- A list of techs / Frameworks including versions. -->
+
+Example
+* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [Maven](https://maven.apache.org/) - Dependency Management
+* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
+
+### Project Code Style
+<!-- Discribe the style used inside the project -->
+
+Example
+This project follows StandardJS "JavaScript Standard Style" coding style conventions. Visit JavaScript Standard Style for a complete list of coding style rules followed in this project.
+
+### Linters and tools
+<!-- List and explain how to use Linter tools -->
+
+### Prerequisites
+<!-- A list of development machine requirements to be able to run this project. -->
+
+What things you need to install the software and how to install them
+
+```
+Give examples
+```
+
+## Getting Started 
+<!-- These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system. -->
+
+### Installing
+<!-- A step by step series of examples that tell you how to get a development env running -->
+
+Say what the step will be
+
+```
+Give the example
+```
+
+And repeat
+
+```
+until finished
+```
+
+## Deployment
+<!-- Add additional notes about how to deploy this on a live system -->
+
+## Troubleshoting
+<!-- If found, list possible solutions to common problems -->
+
+## Third Party plugins and libraries
+<!-- List of external tools used in the project -->
+
+### FrontEnd
+### BackEnd
+
+## Scripts and Helper functions
+<!-- List and explain how to run them inside the project -->
+
+## Filters and Hooks
+<!-- List and explain how to run them inside the project -->
+
+## Running the tests and QA tools
+<!-- Explaining how to run test and where to add tests -->
+
+Example
+1. To run the automated testing system you may run:
+
+```
+npm test
+```
+
+Launches the test runner in the interactive watch mode.<br> See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### Break down into end to end tests
+<!-- Explain what these tests test and why -->
+
+## API Reference
+<!-- Depending on the size of the project, if it is small and simple enough the reference docs can be added to the README. For medium size to larger projects it is important to at least provide a link to where the API reference docs live. -->
+
+## Versioning
+<!-- Description about Semantic Versioning and how and when you have to update the version of the code for future references. -->
+
```

Comment:
Looks good, Oscar. Maybe something about what needs to be in .gitignore too?

Reply:
This could be tricky because it's a general readme, it could be a react or wordpress. So the files would be different. The solution I propose is to make sample gitignores for different types of projects. What do you think? @arzola @bethshook

Reply:
@katialira looks goood